composer.json

```diff
@@ -27,7 +27,12 @@
         "symfony/twig-bundle": "^2.3 || ^3.0"
     },
     "require-dev": {
+        "doctrine/couchdb": "^1.0.0@dev",
+        "doctrine/couchdb-odm": "^1.0.0@dev",
+        "doctrine/couchdb-odm-bundle": "^2.1.0@dev",
         "doctrine/doctrine-bundle": "^1.3",
+        "doctrine/orm": "^2.5.5",
+        "doctrine/mongodb-odm": "^1.1.1",
```

Comment:
this makes it impossible for users to install PHPUnit locally if they don't have the mongo extension (and it does not exist on PHP 7), so this looks like a bad idea to me: it makes collaboration harder